extractor/src/test/resources/com.puppycrawl.tools.checkstyle.checks.whitespace.methodparampad.InputFile/InputMethodParamPadWhitespaceAround.java

```diff
@@ -0,0 +1,48 @@
+/*
+MethodParamPad
+allowLineBreaks = (default)false
+option = SPACE
+tokens = (default)CTOR_DEF, LITERAL_NEW, METHOD_CALL, METHOD_DEF, SUPER_CTOR_CALL, \
+         ENUM_CONSTANT_DEF, RECORD_DEF, RECORD_PATTERN_DEF
+
+
+*/
+
+package com.puppycrawl.tools.checkstyle.checks.whitespace.methodparampad;
+
+@SuppressWarnings({"this", "that"})
+public class InputMethodParamPadWhitespaceAround
+{
+    protected InputMethodParamPadWhitespaceAround ( int i )
+    {
```

Comment:
Remove whole content of class, it should be empty class.

Reply:
done